### PR TITLE
Add missing Uzbek Language Support

### LINKED ISSUE #1588: 🚀 Feature: Add i18n - Uzbek Language Support
### 🔖 Feature description

People in Uzbekistan speak in Uzbek so this feature is to help them.

### 🎤 Why is this feature needed ?

This feature is needed for the people who don't know english but know Uzbek.

### ✌️ How do you aim to achieve this?

- Add a new translation file here: packages/notification-center/src/i18n/languages. You can follow the en.ts example.
- Add your translation import to the TRANSLATIONS object at packages/notification-center/src/i18n/lang.ts
- Add the language code to I18NLanguage type at lang.ts file.
- Updating /docs/docs/notification-center/react-components.md for the language added.
- Submit a PR

### 🔄️ Additional Information

_No response_

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Smartmind12**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature to add i18n - Uzbek Language Support

- **Why was this change needed?** (You can also link to an open issue here)
Fixes #1588 

- **Other information**:
Please Review and Merge the same

### CODE REVIEW

File: packages/notification-center/src/i18n/languages/uz.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const UZ: ITranslationEntry = {
+  translations: {
+    notifications: 'bildirishnomalar',
+    markAllAsRead: 'Hammasini o'qing deb belgilang',
+    poweredBy: 'Tomonidan qo'llab-quvvatlanadi',
+    settings: 'sozlamalar',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/uz.ts:8]:
```suggestion
    notifications: 'Bildirishnomalar',
    markAllAsRead: 'Hammasini o'qing deb belgilang',
    poweredBy: 'Tomonidan qo'llab-quvvatlanadi',
    settings: 'Sozlamalar',
```
Please capitalise settings and notifications translations.

**Review by p-fernandez — APPROVED** [review]

File: packages/notification-center/src/i18n/languages/uz.ts (hunk 1 above)
Comment (Smartmind12) [reply packages/notification-center/src/i18n/languages/uz.ts]: @p-fernandez Thanks I have updated the changes!

File: packages/notification-center/src/i18n/languages/uz.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const UZ: ITranslationEntry = {
+  translations: {
+    notifications: 'Bildirishnomalar',
+    markAllAsRead: 'Hammasini o'qing deb belgilang',
+    poweredBy: 'Tomonidan qo'llab-quvvatlanadi',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/uz.ts:7]:
```suggestion
    poweredBy: `Tomonidan qo'llab-quvvatlanadi`,
```

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: packages/notification-center/src/i18n/languages/uz.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    settings: 'Sozlamalar',
+  },
+  lang: 'uz',
+};
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/uz.ts:11]:
```suggestion
export const UZ: ITranslationEntry = {
  translations: {
    notifications: 'Bildirishnomalar',
    markAllAsRead: `Hammasini o'qing deb belgilang`,
    poweredBy: `Tomonidan qo'llab-quvvatlanadi`,
    settings: 'Sozlamalar',
  },
  lang: 'uz',
};
```

Comment (p-fernandez) [reply packages/notification-center/src/i18n/languages/uz.ts:11]: We need to avoid the unterminated string literals.

**Review by p-fernandez — APPROVED** [review]

**Comment by Smartmind12** [comment]
@p-fernandez can you help me figure out what is missing because of which the branch is not merging into the main branch??

**Pull request merged**